windows/scripts/extract_kernels.py

```diff
@@ -241,13 +265,38 @@ def is_valid_kernel(path):
         b = fp.read(2)
         return b == 'MZ'
 
+def get_files_in_dir(dir):
+    ret = []
+
+    for root, _, files in os.walk(dir):
+        for f in files:
+            filepath = os.path.join(root, f)
+            ret.append(filepath)
+
+    return ret
+
 def extract_kernels_from_container(output_dir, container, files):
+    container_is_dir = False
+
+    if os.path.isdir(container):
+        if files:
+            raise Exception('Files must be null if container is a folder')
```

Comment:
What does "null" mean here? Null is not really a thing in Python - do you mean `None`? Or an empty list?